src/Microsoft.AspNetCore.Routing.Abstractions/Address.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Reflection;
+
+namespace Microsoft.AspNetCore.Routing
+{
+    public class Address
+    {
+        public Address(string name)
```

Comment:
Do you think that addresses are polymorphic? (should this class be abstract or sealed)